### PR TITLE
Setup dependencies, webpack, server, basic stateful react component

### PR DESCRIPTION
**Pull request by John-Michael-Long**
[trello ticket](https://trello.com/c/Ok5ka4ai/10-setup-dependencies-webpack-server-basic-stateful-react-component)

### CODE REVIEW

**Comment by John-Michael-Long** [comment]
@zmattingly can you review my PR

File: client/index.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import GameList from './components/GameList.jsx';
+import data from './dummyData.js';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: null
```
Comment (zmattingly) [inline client/index.jsx:11]: Is this 'value' property used anywhere currently?

File: client/index.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+/*
+// Colors:
+// rgba( 0, 0, 0, 0.2 );
+// off hover: 
+// title text color: #c7d5e0;
+// categories & platform icons text color, on hover: #384959;
+
+// linear-gradient( to bottom, rgba(42,71,94,1.0) 5%, rgba(42,71,94,0.0) 70%)
+
+*/
```
Comment (zmattingly) [inline client/index.jsx:36]: May want to put these notes somewhere else like a separate .md file.

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        query: {
+          presets: ['react', 'env']
+        },
+      },
+    ],
+  },
+};
+
+
+// module.exports = {
```
Comment (zmattingly) [inline webpack.config.js:28]: I'd remove these commented-out lines if you're not using them.

**Review by zmattingly — COMMENTED** [review]
Looks good, just a few minor things for cleanup I've noted.

**Pull request merged**